go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -1610,6 +1612,16 @@ func TestTypes(t *testing.T) {
 			`type:ROW row_event:<table_name:"vitess_decimal" row_changes:<after:<lengths:1 lengths:7 lengths:7 values:"2-1.2300-1.2300" > > > `,
 			`gtid`,
 			`commit`,
+		}, {
+			`begin`,
+			`type:ROW row_event:<table_name:"vitess_decimal" row_changes:<after:<lengths:1 lengths:6 lengths:6 values:"31.23001.2300" > > > `,
```

Comment:
I guess its because I do not understand the encodings, but `31.23001.2300` is mightily confusing to me. What separates the values?

Reply:
Ah, skip that. solved by `lengths`.